composer.json

```diff
@@ -0,0 +1,36 @@
+{
+  "name": "harvesthq/chosen",
```

Comment:
To be sure; can the name be an arbitrary value? Because this won't match with the repo path `harvesthq/chosen-package`.

Reply:
Yeah, I think that can be.